internal/test/client/preview.go

```diff
@@ -69,3 +71,11 @@ func (c *FakePreviewsClient) Destroy(_ context.Context, _ string) error {
 func (*FakePreviewsClient) ListEndpoints(_ context.Context, _ string) ([]types.Endpoint, error) {
 	return nil, nil
 }
+
+func (c *FakePreviewsClient) SleepPreview(_ context.Context, _ string) error {
+	return c.response.ErrSleepPreview
+}
+
+func (c *FakePreviewsClient) WakePreview(_ context.Context, _ string) error {
+	return c.response.ErrWakePreview
+}
```

Comment:
```suggestion

```
do we really need these lines in this PR? I cannot see any interface modification adding this new behavior so test shouldn't need this.